src/uwtools/api/file.py

```diff
@@ -36,7 +35,7 @@ def copy(
     """
     copier = Copier(
         target_dir=Path(target_dir) if target_dir else None,
-        config=_ensure_data_source(_str2path(config), stdin_ok),
+        config=_ensure_data_source(config, stdin_ok),
```

Comment:
The `str2path()` conversion is now provided in `Copier`, so we don't need to do it here. Ditto in the `Linker` constructor call below.